src/cockpit/channels/http.py

```diff
@@ -108,11 +108,19 @@ def request(self):
                            self.options.get('path'),
                            headers=self.options.get('headers') or {},
                            body=self.body)
-        response = connection.getresponse()
-        self.loop.call_soon_threadsafe(lambda: self.send_control(
-            command='response', status=response.status, reason=response.reason,
-            headers=self.parse_headers(response.headers)))
-        self.read_send_response(response)
+        try:
+            response = connection.getresponse()
+            self.loop.call_soon_threadsafe(lambda: self.send_control(
+                command='response', status=response.status, reason=response.reason,
+                headers=self.parse_headers(response.headers)))
+            self.read_send_response(response)
+        except http.client.HTTPException as error:
+            logger.debug('HTTP %s reading response failed: %s', error)
```

Comment:
This passes one argument while the format string expects two? Also shouldn't we call `str(error`)